app/models/merchant.rb

```diff
@@ -17,4 +18,21 @@ def toggle_status
   def self.group_by_status(status)
     self.where(status: status)
   end
+
+  def self.top_five
```

Comment:
you make it look easy, sir!!
<img src="https://media1.giphy.com/media/7A7iNCHhPpqwaNRgV1/giphy.gif"/>